src/components/manage/Sharing/Sharing.jsx

```diff
@@ -159,6 +164,13 @@ class SharingComponent extends Component {
   UNSAFE_componentWillReceiveProps(nextProps) {
     if (this.props.updateRequest.loading && nextProps.updateRequest.loaded) {
```

Comment:
I don't remember that perfectly at this time, but is there a possibility that the nextProps are ``loaded`` but with an error on the request response? That should be checked if it's the case.

Reply:
If the request failed in that case loading is set to false not the loaded true. You can see the code here
https://github.com/plone/volto/blob/master/src/reducers/sharing/sharing.js#L82

Reply:
Awesome, thanks for checking this for me, I was in a bit of a rush but didn't want to forget this :)